src/Db/AbstractDb.php

```diff
@@ -23,6 +24,8 @@
  */
 abstract class AbstractDb extends AbstractValidator
 {
+    use AdapterAwareTrait;
```

Comment:
This is of no use unless you also implement `AdapterAwareInterface`. I'll add that when merging.